settings.php

```diff
@@ -10,10 +10,32 @@
  * Most sites can configure their database by entering the connection string
  * below. If using master/slave databases or multiple connections, see the
```

Comment:
I thought that we had removed all instances of these words from our codebase with https://github.com/backdrop/backdrop-issues/issues/4451. Can we please change these to `/replica` here?
```suggestion
 * below. If using primary/replica databases or multiple connections, see the
```

Reply:
Changed to primary/secondary as that is the terminology used in the docs page.